docs/docker.md

```diff
@@ -0,0 +1,142 @@
+
+## Supported tags
+
+\* denotes an image that is planned but has not yet been released.
+
+### Alpine Linux
+
+- main, latest
+- 0.2.4-alpine, alpine, 0.2.4
+
+### Debian Linux
+
+- 0.2.4-debian, debian
+
+## Deploying this image
+
+### App configuration
+
+Configuration of the PowerDNS Admin app may be achieved through a mixture of two approaches. With either approach you choose, you will need to be aware of the various settings available for the app.
+
+[PowerDNS Admin Settings](https://github.com/ngoduykhanh/PowerDNS-Admin/blob/master/docs/settings.md)
+
+#### Approach #1
+
+You may pass app settings as environment variables to the container. These environment variables will be automatically inserted into the app.config object. Any environment variable that begins with PDA_ or PDAC_ and is entirely capitalized, will be inserted into the app.config object.
+
+**One key difference** between the two variable prefixes is that variables prefixed with PDAC_ will have the PDAC_ prefix removed before being inserted into the app.config object.
+
+For example, say you pass the environment variable PDA_BIND_ADDRESS with the value 0.0.0.0 to the container. 
+
+    PDA_BIND_ADDRESS=0.0.0.0
+
+This will create a key named PDA_BIND_ADDRESS in app.config with the value of the environment variable value of 0.0.0.0.
+
+Similar to how secrets work for Docker Swarm containers, you may append _FILE to your environment variables to automatically load the contents of the given file path into an environment variable.
+
+**One key difference** here from Docker Swarm secrets is that you provide an absolute file path to any path of our choosing. Also, once the contents of the file are loaded, an environment variable of the same name without the trailing _FILE is created with the contents of the loaded file as the value. The original environment variable with the trailing _FILE is removed.
+
+The following example illustrates a scenario where you want to set the PDA_PDNS_API_KEY app setting in a secure fashion. You have the secret API key stored at /run/secrets/PDA_PDNS_API_KEY.
+
+    PDA_PDNS_API_KEY_FILE=/run/secrets/PDA_PDNS_API_KEY
+
+This would be the equivalent of running:
+
+    export PDA_PDNS_API_KEY=CONTENTS_OF_SECRET_FILE
+
+#### Approach #2
+
+With this approach, you may create traditional PowerDNS Admin config files and map them to an accessible location inside the container. Then you can pass an environment variable to indicate where to load the file from.
+
+For example, say you have mapped a python configuration file to /srv/app/configs/my-config.py in the container. You would add the following environment variable to your deployment configuration:
+
+    FLASK_CONF=/srv/app/configs/my-config.py
+
+**!!! NOTICE !!!** - There is a very specific order in which different sources are loaded. Take careful notice to avoid confusion!
+
+- The config file **/srv/app/configs/default.py** is loaded.
```

Comment:
Can you precise it is empty now.

Reply:
@jbe-dw can you elaborate? I'm thinking a typo may have caused some confusion in reading your comment.